docs/core/porting/project-structure.md

```diff
@@ -0,0 +1,118 @@
+
+* [**Combine existing projects and .NET Core projects into single projects**][option-xproj]
+  
+  *What this is good for:*
+  * Simplifying your build process by compiling a single project rather than compiling multiple projects, each targeting a different .NET Framework version or platform.
+  * Easily generating a NuGet package for consumption.
+  * Allows you to write code for a specific .NET Framework version in your libraries through the use of compiler directives.
+  
+  *Unsupported scenarios:*
+  * Does not allow developers without Visual Studio 2015 to open existing projects. To support older versions of Visual Studio, the [keeping your project files in different folders](#support-vs) is a better option.
+  * Does not allow you to share your .NET Core library across different project types in the same solution file. To support this, [creating a Portable Class Library](#support-pcl) is a better option.
+  * Does not allow for project build or load modifications that are supported by MSBuild Targets and Tasks. To support this, [creating a Portable Class Library](#support-pcl) is a better option.
+
+* <a name="support-vs"></a>[**Keep existing projects and new .NET Core projects separate**][option-xproj-folder]
+  
+  *What this is good for:*
+  * Continuing to support development on existing projects without having to upgrade for developers/contributors who may not have Visual Studio 2015.
+  * Decreasing the possibility in creating new bugs in existing projects because no code churn is required in those projects.
+
+* <a name="support-pcl"></a>[**Keep existing projects and create Portable Class Libraries (PCLs) targeting .NET Core**][option-pcl]
+
+  *What this is good for:*
+  * Referencing your .NET Core libraries in desktop and/or web projects targeting the full .NET Framework in the same solution.
+  * Supporting modifications in the project build or load process. These modifications could be the inclusion of MSBuild Tasks and Targets in your *.csproj file.
+
+  *Unsupported scenarios:*
+  * Does not allow you to write code for a specific .NET Framework version because compiler directives are not supported.
+
+## Example
+
+Consider the repository below:
+
+![][example-initial-project]
+
+[**Source Code**][example-initial-project-code]
+
+There are several different ways to add support for .NET Core for this repository depending on the constraints and complexity of existing projects which are described below.
+
+### Replace Existing Projects with a Multi-targeted .NET Core Project (xproj)
+
+The repository can be reorganized so that any existing *.csproj files are removed and a single *.xproj is created that targets multiple frameworks.  This is a great option because a single project is able to compile for different frameworks.  It also has the power to handle different compilation options, dependencies, etc. per targeted framework.
```

Comment:
you need to escape the asterisk in order to use it in sentences.
also, I'd change single *.xproj to single *.xproj file